apps/docs/content/components/Tooltip.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Tooltip
+status: Experimental
+source: https://github.com/primer/brand/tree/main/packages/react/src/Tooltip/Tooltip.tsx
+storybook: '/brand/storybook/?path=/story/components-tooltip--default'
+description: Use the tooltip component to display a short message when a user hovers or focuses an interactive element.
+---
+
+import {ScreenFullIcon} from '@primer/octicons-react'
+import {PropTableValues} from '../../src/components'
+
+```js
+import {Tooltip} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+```jsx live
+<Tooltip text="Sample tooltip content">
+    <Button>Action</Button>
+</Tooltip>
+```
+
+### Custom direction
+
+By default, tooltips appear below the element. You can change the direction by passing the `direction` prop with a value of `n`, `e`, `s`, or `w`.
+
+As long as there is enough room in the viewport, the tooltip will appear in the specified direction. If there is not enough room, the tooltip will appear in whatever direction it can place the tooltip without overflowing the viewport.
+
+```jsx live
+<Tooltip text="Sample tooltip content" direction="e">
+    <Button>Action</Button>
+</Tooltip>
+```
+
+### Substitute a visible label
+
+If a ubiquitously understood icon is used in place of a text label, you can use the `type` to provide a text label for assistive technologies such as screen readers.
+
+Use this with caution. It is generally better to use a visible label for better usability and accessibility.
+
+```jsx live
+<Tooltip text="Go fullscreen" type="label">
+    <Button hasArrow={false}><ScreenFullIcon /></Button>
+</Tooltip>
+```
+
+## Component props
+
+| name         | type                                                              | default         | required | description                             |
+| ------------ | ----------------------------------------------------------------- | --------------- | -------- | --------------------------------------- |
+| `direction`  | <PropTableValues values={['n', 'e', 's', 'w']} />                 | `'s'`           | `false`  | The text to be displayed in the tooltip |
+| `text`       | `string`                                                          | `undefined`     | `true`   | Determines whether the ActionMenu is open by default |
```

Comment:
Think these are the wrong way around, and one is referencing a different component.

```suggestion
| `direction`  | <PropTableValues values={['n', 'e', 's', 'w']} />                 | `'s'`           | `false`  | The side the tooltip caret appears on |
| `text`       | `string`                                                          | `undefined`     | `true`   |  The text to be displayed in the tooltip |
```